<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>parking_fee</class>
 <widget class="QWidget" name="parking_fee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>parking_fee</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>215</width>
     <height>85</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>缴费</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="new_pushButton">
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="exit_pushButton">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="is_paid_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>193</width>
     <height>330</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>年</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="year_lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="month_lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>应付停车费</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="due_payment_lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>付费时间</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="paid_dateEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>员工_id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="staff_id_lineEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>车位号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="parking_id_comboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>parking_fee</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
